### PR TITLE
refactor(locale): remove CS entry from language menu picker

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes
| Tickets          | MANAGER-6349
| License          | BSD 3-Clause

## Description

Approximate release date: 📆 04/03/2021

### :bug: Bug Fixes

1c7d08e - fix(language.menu): remove force display of cs_CZ language

### :boom: Breaking Changes

f9463f9 - refactor(locale): remove LT entry from language menu picker

### :recycle: Refactor

97c7b0e - refactor: remove Messages_cs_CZ.json files

### :arrow_up: Upgrade

5c16003 - fix: bump semver range for @ovh-ux/manager-config package
e365bde - fix: bump semver range for @ovh-ux/manager-config

### :construction_worker_man: Build

9147458 - build: remove extra languages from ContextReplacementPlugin

### :link: Related

- #4229
- #4246
- #4365

### CODE REVIEW

File: packages/manager/apps/dbaas-logs/webpack.config.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ module.exports = (env = {}) => {
     plugins: [
       new webpack.ContextReplacementPlugin(
         /moment[/\\]locale$/,
-        /cs|de|en-gb|es|es-us|fi|fr-ca|fr|it|lt|pl|pt/,
+        /de|en-gb|es|es-us|fr-ca|fr|it|pl|pt/,
```
Comment (cbourgois) [inline packages/manager/apps/dbaas-logs/webpack.config.js:59]: 👍

**Review by cbourgois — APPROVED** [review]

**Review by ganeshkumar1989 — APPROVED** [review]

File: packages/manager/modules/billing/package.json [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
   "dependencies": {
-    "@ovh-ux/manager-config": "^3.0.0 || ^4.0.0",
+    "@ovh-ux/manager-config": "^4.0.0 || ^5.0.0",
     "@ovh-ux/manager-models": "^1.4.3",
     "@ovh-ux/ufrontend": "^1.0.0",
     "lodash-es": "^4.17.15"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-config": "^3.0.0 || ^4.0.0",
+    "@ovh-ux/manager-config": "^4.0.0 || ^5.0.0",
```
Comment (marie-j) [inline packages/manager/modules/billing/package.json:21]: Declared twice ?

File: packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ module.exports = ({ name, description }) => ({
     'start:watch': `lerna exec --stream --parallel --scope='@ovh-ux/manager-${name}-app' --include-dependencies -- npm run dev:watch --if-present`,
   },
   dependencies: {
-    '@ovh-ux/manager-config': '^3.0.0 || ^4.0.0',
+    '@ovh-ux/manager-config': '^4.0.0 || ^5.0.0',
```
Comment (marie-j) [inline packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js:22]: No webpack config generated by sao ? Or not needed to patch the language list ?

**Review by marie-j — APPROVED** [review]

File: packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js (hunk 3 above)
Comment (antleblanc) [reply packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js:22]: There's already a webpack configugration located into the template `folder`.
see: https://github.com/ovh/manager/tree/master/packages/manager/tools/sao-ovh-manager-app/template

From this bare minimum configuration we didn't opt-in to the usage of `ContextReplacementPlugin`.

If you think that it must be part of the default plugins list, then maybe we can think about our default shared webpack configuration where we defined plugins usage.

**Comment by antleblanc** [comment]
Rebasing on `master` branch (got 3 approvals).

**Comment by antleblanc** [comment]
Rebased successfully on `release/2021-w09` branch.

**Pull request merged**